### DISCUSSION TITLE
Cannot open plugdata macOS Sequoia version 15.1.1 M1 Pro

### DISCUSSION THREAD

**Discussion by mengvenk** [plugdata-team/plugdata#1986] [Q&A] [answered]
Hey, I can't open plugdata on my MacBook. Despite reinstalling the latest version, I am still unable to open it. How to fix the problem ?

**Comment by timothyschoen** [comment]
Hi! Have you tried the latest nightly build? They are available on the website, at the bottom of the page:

https://plugdata.org/download.html

**Reply by mengvenk** [reply, marked as answer]
I’ve noticed that the nightly version takes a very long time to download from your official website. So, I downloaded the old version 0.9.0, and fortunately, it works well.
